atom/browser/api/atom_api_window.cc

```diff
@@ -858,6 +864,7 @@ void Window::BuildPrototype(v8::Isolate* isolate,
       .SetMethod("unhookWindowMessage", &Window::UnhookWindowMessage)
       .SetMethod("unhookAllWindowMessages", &Window::UnhookAllWindowMessages)
       .SetMethod("setThumbnailClip", &Window::SetThumbnailClip)
+      .SetMethod("setThumbnailTooltip", &Window::SetThumbnailTooltip)
```

Comment:
@kevinsawicki Was referring to capitalizing (the second T in ToolTip) `"setThumbnailToolTip"`